dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/basic/docker-compose.yaml

```diff
@@ -32,6 +32,22 @@ services:
       interval: 5s
       timeout: 60s
       retries: 120
+    depends_on:
+      - hdfs
```

Comment:
This only guarantees service start up order, and it might make the tests flaky, please use dependent condition `service_healthy: healthy`